internal/blueprint/blueprint.go

```diff
@@ -101,6 +101,18 @@ func (b *Blueprint) Initialize() error {
 		return fmt.Errorf("Error hashing passwords: %s", err.Error())
 	}
 
+	for i, p := range b.Packages {
+		if len(p.Name) == 0 {
+			var errMsg string
+			if len(p.Version) == 0 {
+				errMsg = fmt.Sprintf("Entry #%d has neither version nor name.", i+1)
```

Comment:
No version is probably fine, isn't it? I don't remember ever putting a version into a blueprint... Thus, I would remove this, because it suggests that the version is a required field.

Reply:
It's just "no version" AND "no name" - I'll improve the wording